sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpProducer.cs

```diff
@@ -574,6 +579,18 @@ public override async Task CloseAsync(CancellationToken cancellationToken)
             return link;
         }
 
+        /// <summary>
+        ///   Determines if measuring a sequence number is required to accurately calculate
+        ///   the size of an event.
+        /// </summary>
+        ///
+        /// <param name="activeFeatures">The set of features which are active for the producer.</param>
+        ///
+        /// <returns><c>true</c> if a sequence number should be measured; otherwise, <c>false</c>.</returns>
+        ///
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
just curious - what criteria determines whether or not this attribute should be used?

Reply:
There's no objective criteria that I'm using.  In this case, these are very simple methods that I'd consider writing as a simple statement that repeats in a few places, but wanted to avoid that repetition.   Since they're stateless, simple, and used in a couple of places, hinting to the compiler seemed reasonable.  To be transparent, I did not notice any difference in timings with or without them, so I'm guessing the compiler is optimizing the same regardless (or these are so trivial that the jumping isn't a meaningful measure).